app/main.py

```diff
@@ -1,2 +1,94 @@
+from typing import Any
+
+
+class Node:
+    def __init__(self, key: Any, hash_key: int, value: Any) -> None:
```

Comment:
The correct type annotation for key is Hashable, check everywhere

Reply:
I don't understand what you want